src/Integration.Vsix/packages.config

```diff
@@ -57,7 +57,7 @@
   <package id="System.Diagnostics.Tools" version="4.3.0" targetFramework="net46" />
   <package id="System.Dynamic.Runtime" version="4.3.0" targetFramework="net46" />
   <package id="System.Globalization" version="4.3.0" targetFramework="net46" />
-  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net46" />
+  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net45" />
```

Comment:
Setting this to `net45` probably doesn't make any difference, but doesn't do any harm.